packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -1271,6 +1286,12 @@ export class AsgCapacityProvider extends Construct {
     this.machineImageType = props.machineImageType ?? MachineImageType.AMAZON_LINUX_2;
     this.canContainersAccessInstanceRole = props.canContainersAccessInstanceRole;
     this.enableManagedTerminationProtection = props.enableManagedTerminationProtection ?? true;
+    this.enableManagedDraining = props.enableManagedDraining;
+
+    let managedDraining = undefined;
+    if (this.enableManagedDraining != undefined) {
+      managedDraining = this.enableManagedDraining ? 'ENABLED' : 'DISABLED';
+    }
```

Comment:
nit: I wonder is there any differences when `managedDraining` is `underfined` or `DISABLED`? Can we set it to `DISABLED` first and change to `ENABLED` when `enableManagedDraining` is set?